src/exercise/03.md

```diff
@@ -2,7 +2,17 @@
 
 ## 📝 Your Notes
 
-Elaborate on your learnings here in `src/exercise/03.md`
+Spread Operator
+
+JSX can be written like
+`<div {...props} />` where props is an object containing children and className, for example.
+
+However, for readability, this is better: 
+```
+const props = {children, className};
```

Comment:
I'm not sure what this `props` declaration is doing? For the markup below you'd just need `children` and `className` without putting them in an object.

(I know this is just a note but I want to be sure I'm not missing something ...) :)

Reply:
oh good point!  no, you aren't missing anything.  I had the markup different before.